.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
this should not be here. In `main` this file is ignored via `.gitignore`

Reply:
wait, forgot about this, this should not be part of the PR. So deleting file and commit the deletion